fdbclient/ManagementAPI.actor.cpp

```diff
@@ -1548,10 +1548,10 @@ ACTOR Future<int> setDDMode( Database cx, int mode ) {
 	}
 }
 
-ACTOR Future<std::set<NetworkAddress>> checkForExcludingServers(Database cx, vector<AddressExclusion> excl,
-                                                                bool waitForAllExcluded) {
+ACTOR Future<std::set<IPAddress>> checkForExcludingServers(Database cx, vector<AddressExclusion> excl,
```

Comment:
Multiple SSes can have the same IP but different ports.  So IPAddress alone is not enough to distinguish them.
We may use `AddressExclusion` to replace `NetworkAddress `.

Reply:
I think there's another bug here where if you exclude a machine without specifying an ip, it will incorrectly claim that the machine can be safely removed, since non of the excluding servers (which will all have non-zero ports) will compare equal to the exclusion. I'm going to take a different approach here and use `addressExcluded` to match exclusions and addresses instead of `==`